mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -1115,4 +1145,57 @@
     <string name="search_edit_custom_engine_success_message">Saved %s</string>
     <!-- Text shown when a user successfully deletes a custom search engine -->
     <string name="search_delete_search_engine_success_message">Deleted %s</string>
+
+    <!-- Description text for the Firefox brand logo -->
+    <string name="firefox_logo_description">Firefox logo</string>
+    <!-- Title text shown for the migration screen to the new browser -->
+    <string name="migration_title">Welcome to an all-new Firefox</string>
+    <!-- Description text followed by a list of things migrating (e.g. Bookmarks, History) -->
+    <string name="migration_description">A completely redesigned browser awaits, with improved performance and features to help you do more online.\n\nPlease wait while we update Firefox with your</string>
+    <!-- Text on the disabled button while in progress -->
+    <string name="migration_updating_app_button_text">Updating %s…</string>
+    <!-- Text on the enabled button -->
+    <string name="migration_update_app_button">Start %s</string>
+    <!-- Accessibility description text for completed migration item (e.g. Bookmarks, History) -->
+    <string name="migration_icon_description">Migration status: %s</string>
+
+    <!-- Site Permissions Preferences -->
+    <!-- Preference for reducing image sizes, bandwidth and platform-level optimizations -->
+    <string name="preference_optimize">Optimize</string>
+    <!-- Label summary to explain how the optimize preference works -->
+    <string name="preference_optimize_summary">Lower image quality, throttle streaming bandwidth, and platform-level optimizations</string>
+    <!-- Preference for applying recommend rules to all sites -->
+    <string name="preference_recommended_settings">Use recommended settings</string>
+    <!-- Label summary to explain how the recommended settings work -->
+    <string name="preference_recommended_settings_summary"><![CDATA[
+        <b>Blocked</b>
+
+        <br/> Ads, autoplay sound and video (block playing media with sound), cookies (block third-party trackers cookies), trackers (allow some trackers), pop-up, website redirects. <br/><br/> <b>Ask to allow</b>
+
+        <br/> Camera, location, microphone and notification. <br/><br/> <b>Allowed</b>
+
+        <br/> DRM audio and video, JavaScript, cache and site data, images.]]></string>
```

Comment:
The layout code here allows localizers to change it if they want.
Are we sure we want that?